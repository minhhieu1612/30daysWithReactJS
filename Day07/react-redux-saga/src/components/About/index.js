import React, { Component } from 'react'

const About = () => (
  <div className="block-center">
    <h1 style={{ color: 'blue' }}>Page About</h1>
  </div>
)

export default About
