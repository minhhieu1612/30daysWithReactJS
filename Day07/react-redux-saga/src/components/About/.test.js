import React from 'react'
import About from '.'
import renderer from 'react-test-renderer'

it('Component about testing', () => {
  const component = renderer.create(
    <About />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})