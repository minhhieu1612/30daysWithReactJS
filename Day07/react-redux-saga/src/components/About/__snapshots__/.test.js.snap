// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component about testing 1`] = `
<div
  className="block-center"
>
  <h1
    style={
      Object {
        "color": "blue",
      }
    }
  >
    Page About
  </h1>
</div>
`;
